Xamarin.Forms.Xaml/XmlnsHelper.cs

```diff
@@ -46,5 +59,17 @@ public static void ParseXmlns(string xmlns, out string typeName, out string ns,
 					typeName = decl;
 			}
 		}
+
+		static void ParseUsing(string xmlns, out string typeName, out string ns, out string asm, out string targetPlatform)
+		{
+			typeName = ns = asm = targetPlatform = null;
+
+			foreach (var decl in xmlns.Split(';')) {
```

Comment:
silly user gets silly results.

I left the foreach, like in cld-namespace, because at some point in the future someone will surely asks for supporting targetPlatform